src/frontend/citizen-portal/src/app/services/notice-of-dispute.service.ts

```diff
@@ -155,7 +155,7 @@ export class NoticeOfDisputeService {
     return this.countFormConfigs;
   }
 
-  createNoticeOfDispute(input: NoticeOfDispute): void {
+  createNoticeOfDispute(input: any): void {
```

Comment:
revert this

Reply:
Love to, but the code wouldn't compile without changing this. This is because __skip is not a declared member of this class, yet it is referenced and is a member if you put a breakpoint in the code and inspect.

Reply:
But revert this shouldn't have problem if we can parse the `count` correctly

Reply:
This is why I had to use any. Otherwise, for some reason, I get this:
![image](https://github.com/bcgov/jag-traffic-courts-online/assets/55215368/ce2e070a-364a-4006-a44e-f012b1c50e47)

Yet, the property does exist on the interface.
![image](https://github.com/bcgov/jag-traffic-courts-online/assets/55215368/b61f04a5-baec-4292-a93e-8f748c4715fe)

Not sure how to fix this. It should be strongly typed, yet the compiler is not recognizing a property on an Object even though it's there.